formulas/wick-base/explorers/os

```diff
@@ -62,6 +62,19 @@ if [[ -f /etc/arch-release ]]; then
     exit
 fi
 
+# Check os-release files for ID
+if [[ -f /etc/os-release ]]; then
+    . /etc/os-release
```

Comment:
I really don't like sourcing files that are outside of my control.  Can `grep` be used instead?